src/main/java/net/tylers1066/beaming/sign/CrewSign.java

```diff
@@ -24,7 +29,21 @@
 import org.bukkit.event.player.PlayerRespawnEvent;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class CrewSign implements Listener {
+
+    private final Map<String, Craft> respawnCrafts = new HashMap<>();
```

Comment:
Let's use a weak hash map from UUID to craft.  That will make it easier to deal with players joining/leaving/being offline without causing memory leaks.

Reply:
There shouldn't be any memory leaks here, the map entries are associated with names on craft signs, not players directly, and the map will be emptied when all crafts are released. I did it this way because it doesn't require looking up UUIDs of offline players, and the crew signs don't keep track of name changes anyway. Also, iirc WeakHashMap doesn't work well with value-based types like String and UUID.